ATG/api/account_v1.py

```diff
@@ -27,16 +24,17 @@ def get_account_by_riot_id(
     return response
 
 
-@on_predicate(
-    runtime,
-    predicate=lambda r: r.status_code == 429,
-    value=lambda r: int(r.headers.get("Retry-After")),
-    jitter=None,
-)
-@limits(calls=MAX_CALLS_PER_MINUTE, period=ONE_MINUTE)
+# @backoff.on_predicate(
+#     backoff.runtime,
+#     predicate=lambda r: r.status_code == 429,
+#     value=lambda r: int(r.headers.get("Retry-After")),
+#     jitter=None,
+# )
+@backoff.on_predicate(backoff.expo, predicate=lambda r: r.status_code == 429, max_tries=8)
 def get_account_by_puuid(
     puuid: str, api_key: str, routing: str = "americas"
 ) -> Response:
+    time.sleep(200/120)
```

Comment:
Really not ideal but until https://github.com/RiotGames/developer-relations/issues/1050 is handled, this is a "decent" patch. Ideally, we parse the headers long term.